discord-scripts/thread-management/check-thread-archiving.ts

```diff
@@ -37,7 +37,11 @@ const THREAD_CHECK_CADENCE = 12 * HOUR // 12 * HOUR
 // Use a ThreadAutoArchiveDuration as we'll still lean on Discord to
 // auto-archive after issuing the warning, so we want the value to be
 // one that we can update auto-archiving to.
-// const AUTO_ARCHIVE_WARNING_LEAD_MINUTES: ThreadAutoArchiveDuration = ThreadAutoArchiveDuration.OneDay
+const AUTO_ARCHIVE_WARNING_LEAD_MINUTES: ThreadAutoArchiveDuration =
+  ThreadAutoArchiveDuration.OneDay
+
+// Let's grab if it's the weekend, 0 = sunday, 6 = saturday
+const isWeekend = (): boolean => [0, 6].includes(new Date().getDay())
```

Comment:
Also thinking about holidays here, rough stuff. I wonder if we avoid Mondays as well for that reason 🤔